virttest/utils_v2v.py

```diff
@@ -2071,3 +2111,25 @@ def handle_prompts(session, timeout=300, interval=1.0):
         subproc.close()
 
     return cmd_result
+
+
+def compare_md5(file_a, file_b):
```

Comment:
@xiaodwan this function seems could be helpful for other tests. Are we considering moving it to another place so that it could be called without having to import utils_v2v ?